documentation/docs/mcp/github-mcp.md

```diff
@@ -10,66 +10,300 @@ import CLIExtensionInstructions from '@site/src/components/CLIExtensionInstructi
+    ```
 
-    <CLIExtensionInstructions
-      name="github"
-      command="npx -y @modelcontextprotocol/server-github"
-      timeout={300}
-      envVars={[
-        { key: "GITHUB_TOKEN", value: "••••••••••••••••" }
-      ]}
-      infoNote={
-        <>
-          When creating your access token, you can specify the repositories and granular permissions you'd like Goose to have access to.{" "}
-          <a
-            href="https://github.com/settings/personal-access-tokens"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Create one here
-          </a>.
-        </>
-      }
-    />
+    9. Enter your GitHub Personal Access Token
+    ```sh
+        ┌   goose-configure 
+        │
+        ◇  What would you like to configure?
+        │  Add Extension 
+        │
+        ◇  What type of extension would you like to add?
+        │  Remote Extension (Streaming HTTP) 
+        │
+        ◇  What would you like to call this extension?
+        │  github
+        │
+        ◇  What is the Streaming HTTP endpoint URI?
+        │  https://api.githubcopilot.com/mcp/
+        │
+        ◇  Please set the timeout for this tool (in secs):
+        │  300
+        │
+        ◇  Would you like to add a description?
+        │  No
+        │
+        ◇  Would you like to add custom headers?
+        │  Yes
+        │
+        ◇  Header name:
+        │  Authorization
+        │
+        // highlight-start
+        ◆  Header value:
+        │  Bearer ghp_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+        // highlight-end
+        └ 
+    ```
 
+    10. Choose not to add another header
+    ```sh
+        ┌   goose-configure 
+        │
+        ◇  What would you like to configure?
+        │  Add Extension 
+        │
+        ◇  What type of extension would you like to add?
+        │  Remote Extension (Streaming HTTP) 
+        │
+        ◇  What would you like to call this extension?
+        │  github
+        │
+        ◇  What is the Streaming HTTP endpoint URI?
+        │  https://api.githubcopilot.com/mcp/
+        │
+        ◇  Please set the timeout for this tool (in secs):
+        │  100
+        │
+        ◇  Would you like to add a description?
+        │  No
+        │
+        ◇  Would you like to add custom headers?
+        │  Yes
+        │
+        ◇  Header name:
+        │  Authorization
+        │
+        ◇  Header value:
+        │  Bearer ghp_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+        │
+        // highlight-start
+        ◆  Add another header?
+        │  No 
+        │
+        // highlight-end
+        └  Added github extension
+    ```  
   </TabItem>
 </Tabs>
+  
+ :::info Get Your Access Token
+ Replace `<YOUR_GITHUB_PERSONAL_ACCESS_TOKEN>` with your actual token from [GitHub Settings](https://github.com/settings/personal-access-tokens). When creating your access token, you can specify the repositories and granular permissions you'd like Goose to have access to.
+ :::
```

Comment:
hmm im thinking we can remove this. I linked it above in the enter your personal access key section as we do in our past tutorials
```suggestion
```

Reply:
hmm is it possible to do it similar to this?

1. [Launch the installer](goose://extension?cmd=npx&arg=-y&arg=agentql-mcp&id=agentql&name=AgentQL&description=Transform%20unstructured%20web%20content%20into%20structured%20data&env=AGENTQL_API_KEY%3DAgentQL%20API%20Key)
2. Press Yes to confirm the installation
3. Obtain an [AGENTQL_API_KEY](https://dev.agentql.com/api-keys) and paste it in
4. Click Save Configuration
5. Scroll to the top and click Exit from the upper left corner